docs/finalproject/week-3/lesson.md

```diff
@@ -15,3 +21,21 @@ It's important to evaluate our contributions to the group. Here's what we expect
 - A full stack developer must deliver features that touch each part of the stack. This means at minimum you must build a component of UI that interacts with the database and you must configure and manage that entire interaction pathway. At best this means multiple features across different parts of the stack.
 
 For graduation, we evaluate each role against the Skills for the Information Age skills descriptions, which you should check for more details.
+
+
+### Cooling periods during final projects 
+
+Cooling periods are for teams that are struggling to collaborate, communicate or work together in an agile way. This is an indication that the team may not be on track to graduate due to concerns about their professional skills. 
+
+If a team needs a cooling period they will need to: 
+1. Stop working and take a 5 minute break
```

Comment:
The actions here feel like they would take longer than 5 mins?

I think intention is to keep focus on the project, but I think they could spare 15 mins in this situation.